src/nvim/option_defs.h

```diff
@@ -336,6 +336,7 @@ enum {
   STL_ALTPERCENT      = 'P',  ///< Percentage as TOP BOT ALL or NN%.
   STL_ARGLISTSTAT     = 'a',  ///< Argument list status as (x of y).
   STL_PAGENUM         = 'N',  ///< Page number (when printing).
+  STL_SHOWCMD         = 'S',  ///< Showcmd buffer.
```

Comment:
I wonder if dot `.` is semantically closer to what Vim would choose here. Not a blocker though.

Reply:
I'm not sure I get what you mean. You're suggesting `.` instead of `S`?

I did come across some vi stackexchange questions to embed `showcmd` into the [statusline](https://vi.stackexchange.com/questions/2663/embed-showcmd-in-airline) and even [tabline](https://vi.stackexchange.com/questions/16857/showcmd-on-first-line-instead-of-last-line). Perhaps this change has value outside of `'cmdheight' == 0` context. So if we want to get Bram's feedback I guess I could check if he is open to including this(decoupled from `cmdheight`)?

Reply:
> You're suggesting `.` instead of `S`?

yes



> So if we want to get Bram's feedback I guess I could check if he is open to including this(decoupled from `cmdheight`)?

if you're up for it, that's always a worth a check.